source/shared/core_stmt.cpp

```diff
@@ -2218,145 +2223,195 @@ void get_field_as_string( _Inout_ sqlsrv_stmt* stmt, _In_ SQLUSMALLINT field_ind
                 if( is_truncated_warning( state ) ) {
 #endif // !_WIN32  
```

Comment:
we do get truncated warning (the driver try to mimic the same behavior as server cursor). The new commit doesn't check for client buffer anymore.